TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -579,6 +579,32 @@ public static void CallSetCommand(string user, [Group(1)] int minimum)
 	[Command(@"call *count")]
 	public static void CallCountCommand() => IRCConnection.SendMessageFormat("{0} out of {1} calls needed.", TwitchGame.Instance.CallingPlayers.Count, TwitchGame.Instance.callsNeeded);
 
+	[Command(@"uncall")]
+	public static void RemoveQueuedPlayer(string user)
+	{
+		if (!(TwitchGame.Instance.CallingPlayers.Contains(user)))
```

Comment:
```suggestion
		if (!TwitchGame.Instance.CallingPlayers.Contains(user))
```
Minor, just some unnecessary parentheses.